src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,31 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.util.List;
+
 public class WorkWithFile {
```

Comment:
The class name 'WorkWithFile' is not very informative. Consider renaming the class to better reflect its responsibility, such as 'TransactionStatisticsGenerator'.